pkg/model/loader.go

```diff
@@ -151,9 +151,11 @@ func (ml *ModelLoader) ShutdownModel(modelName string) error {
 		return fmt.Errorf("model %s not found", modelName)
 	}
 
+	retries := 1
 	for ml.models[modelName].GRPC(false, ml.wd).IsBusy() {
 		log.Debug().Msgf("%s busy. Waiting.", modelName)
-		time.Sleep(2 * time.Second)
+		time.Sleep(time.Duration(retries * 2) * time.Second)
+		retries++
```

Comment:
since we are at this, better have a maximum cap here, or if we keep failing this potentially could grow indefinitely

Reply:
Personally, I don't usually consider an infinitely growing delay an issue unless it's exponential backoff, but I'll go ahead and put a 2 minute clamp here -- unless you've got a different duration in mind.

Reply:
That's sufficient - thanks!